### PR TITLE
Enable welcome bot

### PR DESCRIPTION
**Pull request by hoechenberger**
As mentioned a while ago in https://github.com/mne-tools/mne-python/issues/8221#issuecomment-738695038,
here's an attempt to enable to "welcome bot" for this repo

https://github.com/apps/welcome

I've granted the bot the requested access rights to our repo (clicked the respective button), but I cannot verify whether this worked out properly, as I don't seem to have permissions to review access rights. 

cc @cbrnr @drammock

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello! 👋 Thanks for opening your first issue here! ❤️ We will try to get back to you soon. 🚴🏽‍♂️
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Hello! 👋  Thanks for opening your first pull request here! ❤️ We will try to get back to you soon. 🚴🏽‍♂️
```
Comment (cbrnr) [inline .github/config.yml:13]:
```suggestion
  Hello! 👋 Thanks for opening your first pull request here! ❤️ We will try to get back to you soon. 🚴🏽‍♂️
```

**Comment by cbrnr** [comment]
Where do I go to see if it is enabled?

**Comment by hoechenberger** [comment]
> Where do I go to see if it is enabled?

I believe the only way to see it is by inspecting the permissions for third-party apps, which I cannot access for this repo

**Comment by cbrnr** [comment]
Can you point me to where these settings are located?

**Comment by hoechenberger** [comment]
> Can you point me to where these settings are located?

I suppose it would be listed under https://github.com/mne-tools/mne-python/settings/installations

**Comment by hoechenberger** [comment]
It appears at least @agramfort has received an email from GH, stating that I have requested the installation of "Welcome". Just wanted to clarify that this email is related to the (attempted) activation of the welcome bot.

**Comment by cbrnr** [comment]
I don't see anything in the settings related to access permissions of this bot.

**Comment by hoechenberger** [comment]
@cbrnr Maybe it didn't work at all, then, because I lack sufficient access rights.

Could you try going to https://github.com/apps/welcome and clicking on "Install"?

**Comment by cbrnr** [comment]
OK done. I can now see it in "Installed GitHub apps".

**Comment by hoechenberger** [comment]
> OK done. I can now see it in "Installed GitHub apps".

Now all we have to to is sit quietly and wait until a new first-timer shows up!!!

**Comment by hoechenberger** [comment]
> Now all we have to to is sit quietly and wait until a new first-timer shows up!!!

Oh. and merge this PR, of course. ;)

**Review by cbrnr — APPROVED** [review]

**Comment by hoechenberger** [comment]
@larsoner @drammock please merge if you think we should try this out 👌

**Comment by larsoner** [comment]
@drammock I'll let you click the button then I'll enable it for the repo

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
thanks @hoechenberger and @cbrnr!

**Comment by larsoner** [comment]
installed

**Pull request merged**